src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -203,6 +218,40 @@ sample({
   target: $accounts,
 });
 
+sample({
+  clock: walletRemoved,
+  source: $accounts,
+  fn: (accounts, wallet) => ({ accounts: accounts.filter((a) => a.walletId === wallet.id), wallet }),
+  target: removeWalletFx,
+});
+
+sample({
+  clock: removeWalletFx.doneData,
+  source: $wallets,
+  fn: (wallets, wallet) => wallets.filter((w) => w.id !== wallet.id),
+  target: $wallets,
+});
+
+sample({
+  clock: removeWalletFx.doneData,
+  source: $accounts,
+  fn: (accounts, wallet) => accounts.filter((a) => a.walletId !== wallet.id),
```

Comment:
Would be interesting to use `spread` from `patronum`
So we receive `walletId`, `fn` returns `{ wallets: wallets.filter..., accounts: accounts.filter }`
`target` gets `spread`
line 197 in this file